src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Link to the stylesheet should have an extension of `.css` instead of `.scss`. SCSS files are preprocessed and should be compiled to CSS before being linked in HTML.